ec/src/models/short_weierstrass_jacobian.rs

```diff
@@ -260,7 +265,6 @@ impl<P: Parameters> Default for GroupAffine<P> {
 #[derivative(
     Copy(bound = "P: Parameters"),
     Clone(bound = "P: Parameters"),
-    Eq(bound = "P: Parameters"),
```

Comment:
Why did this get removed?

Reply:
This got moved to a proper impl, because you shouldn't have a manual impl of `PartialEq` and a derived impl of `Eq`.